DataFormats/MuonReco/interface/Muon.h

```diff
@@ -219,7 +219,11 @@ namespace reco {
       MultiIsoMedium         = 1UL<<26,   // miniIso with ptRatio and ptRel 
       PuppiIsoLoose          = 1UL<<27,
       PuppiIsoMedium         = 1UL<<28,
-      PuppiIsoTight          = 1UL<<29
+      PuppiIsoTight          = 1UL<<29, 
+      MvaVTight              = 1UL<<30,   // > 0.45
+      MvaVVTight             = 1UL<<31,   // > 0.9
+      LowPtMvaLoose          = 1UL<<32,   // > -0.6
+      LowPtMvaMedium         = 1UL<<33,   // > -0.2
```

Comment:
the data member for this is ```unsigned int selectors_;```
So, going to bits 32 (count from 0) and higher will not work.

Reply:
Please don't change the type - we need to stay reasonable with the number of working points. Otherwise it will be unmanageable. 32 bits it should be enough

Reply:
I'm not sure how you can stay with the same 32 bits, because I do not see a clear way how the old bits can be repurposed safely.

Reply:
We will drop unused and re-use/re-purpose existing bits. Each bit is extra work to maintain.